db/silkworm/db/chaindb.cpp

```diff
@@ -524,6 +524,11 @@ int Table::get_prev_dup(MDB_val* key, MDB_val* data) { return get(key, data, MDB
 int Table::get_next(MDB_val* key, MDB_val* data) { return get(key, data, MDB_NEXT); }
 int Table::get_next_dup(MDB_val* key, MDB_val* data) { return get(key, data, MDB_NEXT_DUP); }
 int Table::get_next_nodup(MDB_val* key, MDB_val* data) { return get(key, data, MDB_NEXT_NODUP); }
+int Table::get_next_dup_unrestricted(MDB_val* key, MDB_val* data) {
```

Comment:
Doesn't `get_next` do the same?

Reply:
ops, sorry was just experimenting with it a little bit